src/editors/content/feedback/likertseries/LikertSeriesEditor.tsx

```diff
@@ -30,11 +31,39 @@ export interface Props extends AbstractContentEditorProps<LikertSeries> {
 }
 
 export interface State extends AbstractContentEditorState {
-
+  showGrouping: boolean; // toggle to show item grouping
 }
 
 export class LikertSeriesEditor extends AbstractContentEditor<LikertSeries, Props, State> {
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showGrouping: false,
```

Comment:
showGrouping should default to true if there is at least one LikertItem whose group is not nothing.